charts/nats/values.yaml

```diff
@@ -0,0 +1,221 @@
+###############################
+#                             #
+#  NATS Server Configuration  #
+#                             #
+###############################
+nats:
+  image: nats:2.1.7-alpine3.11
+  pullPolicy: IfNotPresent
+
+  # Toggle whether to enable external access.
+  # This binds a host port for clients, gateways and leafnodes.
+  externalAccess: false
+
+  # Toggle to disable client advertisements (connect_urls),
+  # in case of running behind a load balancer (which is not recommended)
+  # it might be required to disable advertisements.
+  advertise: true
+
+  # In case both external access and advertise are enabled
+  # then a service account would be required to be able to
+  # gather the public ip from a node.
+  serviceAccount: "nats-server"
+
+  # The number of connect attempts against discovered routes.
+  connectRetries: 30
+
+  # How many seconds should pass before sending a PING
+  # to a client that has no activity.
+  pingInterval: 
+
+  # Server settings.
+  limits:
+    maxConnections: 
+    maxSubscriptions: 
+    maxControlLine: 
+    maxPayload: 
+
+    writeDeadline: 
+    maxPending: 
+    maxPings: 
+    lameDuckDuration: 
+
+  logging:
+    debug: 
+    trace: 
+    logtime: 
+    connectErrorReports: 
+    reconnectErrorReports: 
```

Comment:
These do not have values? do they need values or are we using defaults? if we're using defaults maybe comment this out. Perhaps populate the defaults for documentation purposes.

Reply:
@jpweber looks like chart depends on these values:
```
Error: template: astronomer/charts/nats/templates/configmap.yaml:292:20: executing "astronomer/charts/nats/templates/configmap.yaml" at <.Values.nats.limits.maxConnections>: nil pointer evaluating interface {}.maxConnections
```